packit_service/worker/allowlist.py

```diff
@@ -132,6 +132,7 @@ def is_github_username_from_fas_account_matching(self, fas_account, sender_login
             )
             return github_username == sender_login
 
+        logger.debug("github_username not set or account private")
```

Comment:
I wonder if we should log the whole `user_info` here to help with debugging. Or at least `github_username`.

Reply:
I was thinking that logging the whole `user_info` isn't really needed and would just take up too many log lines. `github_username` is logged above if it is not None, I just added this line so that we can be sure when it was None

Reply:
oki, that's true

there is a `is_private` field in the payload:
```python
{'username': 'commonjavabot',
 'surname': 'Bot',
 'givenname': 'Commonjava',
 'human_name': 'Commonjava Bot',
 'emails': ['exd-guild-mpakt@redhat.com'],
 'ircnicks': None,
 'locale': 'en-US',
 'timezone': 'UTC',
 'gpgkeyids': None,
 'sshpubkeys': None,
 'certificates': None,
 'creation': '2017-06-16T16:39:39',
 'is_private': False,
 'locked': False,
 'github_username': 'commonjava-bot',
 'gitlab_username': None,
 'website': None,
 'pronouns': None,
 'rhbzemail': None,
 'uri': 'http://fasjson.fedoraproject.org/v1/users/commonjavabot/'}
```

I'm wondering if we can retrieve the userinfo if the account is private :D

so yeah, let's not log that, we can always check logs or retrieve ourselves